types/masonry-layout/index.d.ts

```diff
@@ -40,6 +40,11 @@ declare class Masonry {
 }
 
 declare namespace Masonry {
+    interface hiddenOrVisibleStyleOption {
```

Comment:
```suggestion
    interface HiddenOrVisibleStyleOption {
```
https://palantir.github.io/tslint/rules/class-name/
I'm not sure if it's enforced here, but pretty much used everhwere